packages/ui/Slottable/useSlots.js

```diff
@@ -77,36 +78,60 @@ function distribute (slots, children) {
 		}
 	}
 
-	return props;
+	// We handle fallback here (rather than at the props initialization) because distributeChild
+	// will append to existing props and we want the distributed value to override the fallback
+	// value.
+	return {
+		...fallback,
+		...props
+	};
 }
 
 /**
  * Configuration for `useSlots`
  *
  * @typedef {Object} useSlotsConfig
  * @memberof ui/Slottable
- * @property {String[]} [slots]    List of slot names
- * @property {Node}     [children] Nodes to distribute into the slots
+ * @property {String[]} [slots] List of slot names
+ * @property {Object}   [props] Props object
  * @private
  */
 
 /**
  * Distributes `children` into the configured `slots`.
  *
+ * `useSlots` iterates over all of the `children` in `props` and distributes any children based on
+ * the followig rules:
+ *
+ * * If the child has a `slot` property matching a valid slot, or
+ * * If the component for the child has the `defaultSlot` static member matching a valid slot, or
+ * * If the child component is a string matching a valid slot.
```

Comment:
This wording is a little unclear.    'child component's type is  a string', maybe?

Reply:
Fixed!